### PR TITLE
Enhance Vietnamese translation

### PR DESCRIPTION
**Pull request by nop33**
Supersedes #527

### CODE REVIEW

**Comment by nop33** [comment]
@Sakrecoer Is it true that the VI translations do not need 3rd party review? In such a case, can I merge this?

**Comment by Sakrecoer** [comment]
Sorry, i know i answered you privately, but for the record: all languages need a 3rd party review onward.

**Comment by duckladyvi** [comment]
The Vietnamese translation form @nit27 is quite good. Everyone speaking Vietnamese would definately understand.

File: locales/vi-VN/translation.json [hunk 1]
```diff
@@ -5,50 +5,50 @@
...
   "Discreet mode": "Ẩn số dư",
   "Default address": "Địa chỉ mặc định",
   "Settings": "Cài đặt",
   "Copied to clipboard!": "Đã sao chép vào clipboard!",
   "Copy to clipboard": "Sao chép vào clipboard",
   "Copied": "Đã sao chép",
-  "Official Wallet": "Ví chính thức",
-  "The easiest way to get started with Alephium.": "Cách đơn giản nhất để bắt đầu với Alephium.",
+  "Official Wallet": "Ví Desktop chính thức",
+  "The easiest way to get started with Alephium.": "Alephium là blockchain Layer 1 đầu tiên giúp mở rộng quy mô và cải thiện trên các công nghệ cốt lõi của Bitcoin, Proof of Less Work & UTXO.",
```
Comment (nop33) [inline locales/vi-VN/translation.json:23]: I am quite sure this is not one-to-one translation of the original text

File: locales/vi-VN/translation.json [hunk 2]
```diff
@@ -163,13 +163,13 @@
   "Enter password": "Nhập mật khẩu",
   "Login": "Đăng nhập",
   "Security Check": "Kiểm tra bảo mật",
-  "Alright! Time to check if you got your words right!": "Tuyệt! Bây giờ hãy kiểm tra nếu bạn ghi chính xác các cụm từ nhé!",
-  "Select the words in the right order. Ready?": "Chọn các từ khôi phục với trình tự chính xác. Sẳn sàng chứ?",
-  "Ready!": "Sẳn sàng!",
+  "Alright! Time to check if you got your words right!": "Tuyệt vời! Bây giờ hãy kiểm tra nếu bạn ghi chính xác các cụm từ nhé!",
+  "Select the words in the right order. Ready?": "Chọn các từ khôi phục với thứ tự chính xác. Sẳn sàng chứ?",
+  "Ready!": "BẮT ĐẦU!",
```
Comment (nop33) [inline locales/vi-VN/translation.json:168]: Does it need to be all in capital?

File: locales/vi-VN/translation.json (hunk 1 above)
Comment (nit27) [reply locales/vi-VN/translation.json:23]: - well, the last Vietnamese version was ambiguous meaning. Therefore, I proposed to add _Desktop_ word into in order to consolidate the context meaning. It means that this is "_The Official Desktop Wallet_", not any kind else. Anyway, if this is not an acceptable in translating unification of the team, please lmk. I will return the basic meaning before. 
- line 23, also the unclear meaning of pure-translation. I proposed to add this quote `Alephium is the first operational sharded L1 blockchain scaling and enhancing PoW & UTXO concepts.` for users easier to understand. I understand it change totally the original meaning of the english version in this line, but Vietnamese people cannot understand when I translated it as pure-translation.

Conclusion, please advice me for the direction. Thanks :-)

File: locales/vi-VN/translation.json [hunk 3]
```diff
@@ -107,7 +107,7 @@
   "Version": "Phiên bản",
   "General": "Tổng quan",
   "Networks": "Mạng",
-  "Wallets": "Ví",
+  "Wallets": "Ví Desktop",
```
[the remark this answers is not in the document]
Comment (nit27) [reply locales/vi-VN/translation.json:110]: Hmm, weird. Let me correct it. Thanks for let me know.

File: locales/vi-VN/translation.json (hunk 2 above)
Comment (nit27) [reply locales/vi-VN/translation.json:168]: My typo, thank you. 
@vynguyen94 I hope you and I could spend time on checking typos like the same way that @nop33 advices us for the next time. 

Thanks in advanced.

**Comment by nit27** [comment]
> Thanks a lot for the translation @nit27 and for the review @vynguyen94! 🙏 ❤️
> 
> I've added some comments, I wonder what your thoughts are.

Thank you @nop33 for having essential questions above, they help me a lot of my contribution. Please let me and @vynguyen94 take a bit time to discuss and improve them as soon as possible. 

Good day! :-)

**Comment by nit27** [comment]
I just made some fixes in ee38fe7. @nop33 you could supersedes this PR, then I  will create a new PR. 
Could you please @vynguyen94 have a look on my fixed translation?

Thanks all.

**Comment by nop33** [comment]
> I just made some fixes in https://github.com/alephium/desktop-wallet/commit/ee38fe77d815780c49ff9c5c9450dc40591fa014. @nop33 you could supersedes this PR, then I will create a new PR.

First create a new PR so that I can get a link to justify the closure of this PR, thanks!

**Comment by nop33** [comment]
Closing in favor of #579

**Pull request closed without merging**